### DISCUSSION TITLE
Import/Export entire DB

### DISCUSSION THREAD

**Discussion by lessborders** [directus/directus#15512] [Archive]
This would be very nice - I have a directus installation on localhost, but for production I am using directus cloud.
There is currently no way to deploy my development changes to production. :(

**Comment by azrikahar** [comment]
I was wondering why the title looked so familiar, and it is actually near identical to #14763! This linked answer still applies for the db import aspect here: https://github.com/directus/directus/discussions/14763#discussioncomment-3299513

In regards to db exports for Standard or Enterprise cloud users, feel free to reach out to the cloud team via the live chat in the cloud dashboard 👍

**Reply by lessborders** [reply]
Only issue is I would have to contact directus every time I want to deploy, which at the moment happens every 2-3 days
